subresource-loading.md

```diff
@@ -201,7 +208,7 @@ Resource-Bundle-Chunk-Ids: 0ijdfs
 
 Existing content from the cache can simply be reused, since the cache is at the granularity of the chunk IDs within the bundle, not at the level of the bundle itself.
 
-### Optionality and URL integrity
+#### Optionality and URL integrity
```

Comment:
This optionality/URL integrity point should be cross-cutting for all variants discussed. It could apply to your doc just as well, I believe.

Reply:
Agree. I thought it was below your example, but I may have miscounted hashes :)